a10_neutron_lbaas/neutron_ext/extensions/a10DeviceInstance.py

```diff
@@ -23,8 +24,6 @@
 
 import a10_neutron_lbaas.neutron_ext.common.constants as constants
 from a10_neutron_lbaas.neutron_ext.common import resources
-import a10_neutronclient.resources.a10_device_instance as a10_device_instance
-
 
 RESOURCE_ATTRIBUTE_MAP = resources.apply_template(a10_device_instance.RESOURCE_ATTRIBUTE_MAP,

```

Comment:
Based on where this is imported from, it looks like resources.apply_template got duplicated between a10_neutron_lbaas and a10_openstack. It should move to a10_opentack_lib at some point.